### ISSUE TITLE
[Feature]: occlusion

### ISSUE THREAD

**Issue by lhucklen** [Gourieff/sd-webui-reactor#566]
### Feature description

Could we see occlusion masking? That feels like it is all this is missing.

**Comment by drdancm** [comment]
This is done very well in FaceFusion

**Comment by Gourieff** [comment]
Yes, I have it in the roadmap but doesn't have free time at the moment to implement this feature
Maybe after NY holidays
